django_elastic_migrations/models.py

```diff
@@ -1157,9 +1162,7 @@ def prepare_action(self, dem_index):
                 raise NoActiveIndexVersion(msg)
 
             if self.newer_mode:
-                versions = self.index.get_newer_versions(given_version=active_version)
-                self.apply_to_newer(versions, UpdateIndexAction())
-                # we're done, because the newer versions will get their own actions
+                self.apply_to_newer(active_version)
```

Comment:
This is the main change: use a single method for repeated `self.apply_to_newer` code. This affects `./manage.py es_update movies --newer`, which creates subtasks for every index that is never than the active index.